<?xml version="1.0"?>
<launch>

  <include file="$(find hiqp_demos)/launch/yumi_simulation.launch"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find grasp_learning)/launch/demo.rviz"/>

  <!-- launch the grasp_learning node -->
  <node name="demo_learning" pkg="grasp_learning" type="demo_learning" respawn="false" output="screen" >
     <param name="with_gazebo" type="bool" value="false"/>
     <param name="generalize_policy" type="bool" value="false"/>
     <param name="decay_rate" type="double" value="0.5"/>
     <param name="exec_time" type="double" value="10"/>
     <param name="task_dynamics" value="TDynPolicy"/>
     <!-- <param name="task_dynamics" value="TDynLinear"/> -->

      <!-- <param name="task" value="gripperToHorizontalPlane"/> -->
      <!--param name="task" value="gripperToVerticalPlane"/-->
      <param name="task" value="gripperToHorizontalAndVerticalPlane"/> 

  </node>
</launch>
